bim2sim/sim_settings.py

```diff
@@ -372,22 +373,34 @@ def update_from_config(self, config):
                         # convert to readable python object
                         try:
                             # todo ast.literal_eval is safer but not safe.
-                            set_from_cfg = ast.literal_eval(set_from_cfg)
+                            set_from_cfg = ast.literal_eval(str(set_from_cfg))
                         except (ValueError, SyntaxError):
-                            pass
-                        # handle Enums (will not be found by literal_eval)
-                        if isinstance(set_from_cfg, str) and\
-                                '.' in set_from_cfg:
-                            enum_type, enum_val = set_from_cfg.split('.')
-                            # convert str to enum
-                            try:
-                                enum_type = getattr(types, enum_type)
-                                val = getattr(enum_type, enum_val)
-                            except AttributeError:
-                                raise AttributeError(
-                                    f" Tried to create the enumeration "
-                                    f"{enum_type} but it doesn't exist.")
+                            logger.warning(f'Failed literal evaluation of '
+                                           f'{set_from_cfg}. Proceeding.')
+                        if isinstance(set_from_cfg, list):
+                            val = set_from_cfg
```

Comment:
is this really working with lists?

Reply:
Ok, but it's redundant with lines 402 ff. Pelase remove this

Reply:
but its duplicate, I'll just remove these two lines.